README.md

```diff
@@ -17,18 +17,6 @@
 $ pip install opensquirrel
 ```
 
-### Editable installation
-
-To perform an editable install, run the following command in the root directory of `OpenSquirrel` with the `-e` flag
```

Comment:
With the `-e` flag is redundant.

Reply:
I would end the phrase with a `:`.

Reply:
I don't see this in code??? The 'Editable installation' section seems to already have been removed. But I don't see any other commits than the ones I made.... Any idea how this could have happened?

Reply:
Ah, OK, sorry, I haven't properly read this comment/question. This section is still in `develop`. If you think it should stay there, just put it back in.

How this could have happened? Maybe you branched from a codebase that had removed it.